chromadb/api/fastapi.py

```diff
@@ -413,15 +414,29 @@ def _add(
         metadatas: Optional[Metadatas] = None,
         documents: Optional[Documents] = None,
         uris: Optional[URIs] = None,
-    ) -> bool:
+    ) -> AddResult:
         """
         Adds a batch of embeddings to the database
         - pass in column oriented data lists
         """
         batch = (ids, embeddings, metadatas, documents, uris)
         validate_batch(batch, {"max_batch_size": self.get_max_batch_size()})
-        self._submit_batch(batch, "/collections/" + str(collection_id) + "/add")
-        return True
+
+        resp_json = self._make_request(
+            "post",
+            "/collections/" + str(collection_id) + "/add",
+            json={
+                "ids": batch[0],
```

Comment:
Since this gets passed to the segment, shouldn't it be possible for `ids` to be `None` here?